modules/unsupported/geojsonstore/src/main/java/org/geotools/data/geojson/GeoJSONWriter.java

```diff
@@ -225,7 +287,8 @@ private Geometry reprojectGeometry(SimpleFeature currentFeature) {
         if (transform == null || inCRS != lastCRS) {
             lastCRS = inCRS;
             try {
-                transform = CRS.findMathTransform(inCRS, outCRS, true);
+                if (inCRS == null) transform = IdentityTransform.create(2);
+                else transform = CRS.findMathTransform(inCRS, outCRS, true);
```

Comment:
I'm surprised that PMD lets you not have braces on this - I think it would be better to have them.

Reply:
PMD does not check them, Google Format should, but yes, it allows a brace-less if/else. Will add (hopefully it does not remove them, have to try)

Reply:
Added braces in the code.